src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -188,5 +212,18 @@ public async Task<bool> CreateDotnetPublishDeploymentBundle(Recommendation recom
 
             return true;
         }
+
+        private async Task LocateCustomRecipePaths(string targetApplicationFullPath)
```

Comment:
Since this looking at for recipes based on `targetApplicationFullPath` the method should not add them to the member level `_recipeDefinitionPaths`. This method should return back the custom recipe paths and then the calling method should create a union of the returned paths and the values from `_recipeDefinitionPaths`.

Reply:
Fixed